test/crd_checks.go

```diff
@@ -30,7 +30,7 @@ import (
 
 const (
 	interval = 1 * time.Second
-	timeout  = 2 * time.Minute
+	timeout  = 4 * time.Minute
```

Comment:
at this point, what do you think about adding a `timeout` argument to the polling functions, so that each test can pass its own timeout? e.g. the helloworld task probably needs < 1 minute, kaniko needs 2 minutes, this one needs 4 minutes. it will help to keep track of why the timeout is what it is, and also will ensure tests don't wait for longer than they need to